rest_api/operator_apis/dns-operator-openshift-io-v1.adoc

```diff
@@ -266,20 +271,20 @@ Required::
 
 | `clusterDomain`
 | `string`
-| clusterDomain is the local cluster DNS domain suffix for DNS services. This will be a subdomain as defined in RFC 1034, section 3.5: https://tools.ietf.org/html/rfc1034#section-3.5 Example: "cluster.local" 
+| clusterDomain is the local cluster DNS domain suffix for DNS services. This will be a subdomain as defined in RFC 1034, section 3.5: https://tools.ietf.org/html/rfc1034#section-3.5 Example: "cluster.local"
  More info: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service
 
 | `clusterIP`
 | `string`
-| clusterIP is the service IP through which this DNS is made available. 
- In the case of the default DNS, this will be a well known IP that is used as the default nameserver for pods that are using the default ClusterFirst DNS policy. 
- In general, this IP can be specified in a pod's spec.dnsConfig.nameservers list or used explicitly when performing name resolution from within the cluster. Example: dig foo.com @<service IP> 
+| clusterIP is the service IP through which this DNS is made available.
+ In the case of the default DNS, this will be a well known IP that is used as the default nameserver for pods that are using the default ClusterFirst DNS policy.
```

Comment:
Use present tense: ".., this is a a well-known" Also, "well-known" is hyphenated.